az-digital/az_quickstart
# Closes #2624: Open specific external links in new tab

## Description

- New branch for the previous issue/2624.
- Includes Chris' changes (not sure if needed now).
- Install new required module with default settings.
- Can have related permissions, but probably better in a separate ticket so specific settings can be exposed to azcontentadmin users rather than all the settings exposed.

## Related issues
Closes #2624 

## How to test

1. Add a PDF to the site
2. Add links to a page
   - PDF on the site
   - Trellis event
   - Google doc
   - Box link
   - OneDrive link
   - Dropbox
3. Test the links to ensure they all open in a new tab/window based on personal setting.

Probo build: https://a5afed90-0208-4d68-9315-65a77c418575.probo.build/

### Example test links

- [Trellis event link](https://events.trellis.arizona.edu/f44lNu67/5a3U3nIwLYK?_gl=1*1ncy4ls*_ga*OTMyMDU0ODAzLjE2OTE0MjM0NjM.*_ga_7PV3540XS3*MTY5NTY3MDQ4NC4zNS4xLjE2OTU2NzA0OTUuNDkuMC4w)
- [Sample Document.pdf](https://17dd72e3-b662-468c-acaf-1d4a01bea572.probo.build/sites/default/files/2023-09/Sample%20Document.pdf)
- [Google doc](https://docs.google.com/document/d/1hYZIm553E7gO_bCxr1K7l0NN_v137k6jNlkUyyzCroY/edit?usp=sharing)
- [Box link](https://arizona.box.com/s/wasg24vebx58n5sv13n9zg68aox59bh0)
- [OneDrive link](https://emailarizona-my.sharepoint.com/:x:/g/personal/danahertzberg_arizona_edu/EcyTb-u8uGpHnmeogdAYRqUBnVZjOPKMxX5v5OB6XuA6Sw?e=KTFRGx)
- [Dropbox](https://www.dropbox.com/scl/fi/8wcc4shset2qkbt3qc6wh/21-UArizona-Dynamic-Template.pptx?rlkey=ms6028rbz9m2nrje7n907o9fg&dl=0)

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
